WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/NewQuickStartChecklistView.swift

```diff
@@ -0,0 +1,218 @@
+import UIKit
+
+final class NewQuickStartChecklistView: UIView, QuickStartChecklistConfigurable {
```

Comment:
[nit] I feel that the prefix "New" is not very descriptive, especially in the long run. I can't really think of a different name though. If you can't think of a better name then we can just add a comment detailing what this view is used for and that it can only handle one quick start tour collection.

Reply:
Good point. Fixed! 4742760